### PR TITLE
[FEAT]: Handling already created key with new or same Ledger device

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Handling alredy created key with new or same Ledger device

https://github.com/user-attachments/assets/88d30e74-8ee9-49bc-a299-904371f08eea

### ❓ Context

- **JIRA or GitHub link**: [LIVE-13805] 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-13805]: https://ledgerhq.atlassian.net/browse/LIVE-13805?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/static/i18n/en/app.json [hunk 1]
```diff
@@ -6449,6 +6450,16 @@
       "cta": "Connect another Ledger",
       "ctaDelete": "Delete my encryption key"
     },
+    "alredySecureError": {
+      "title": "This app is already secured by this Ledger",
+      "description": "If you are attempting to create a new backup, please connect the Ledger device you used to create your other encryption key.",
+      "cta": "I undestand"
```
Comment (thesan) [inline apps/ledger-live-desktop/static/i18n/en/app.json:6456]:
```suggestion
      "cta": "I understand"
```

File: apps/ledger-live-desktop/static/i18n/en/app.json [hunk 2]
```diff
@@ -6449,6 +6450,16 @@
       "cta": "Connect another Ledger",
       "ctaDelete": "Delete my encryption key"
     },
+    "alredySecureError": {
```
Comment (thesan) [inline apps/ledger-live-desktop/static/i18n/en/app.json:6453]:
```suggestion
    "alreadySecureError": {
```

File: apps/ledger-live-desktop/static/i18n/en/app.json [hunk 3]
```diff
@@ -6449,6 +6450,16 @@
...
+      "title": "This app is already secured by this Ledger",
+      "description": "If you are attempting to create a new backup, please connect the Ledger device you used to create your other encryption key.",
+      "cta": "I undestand"
+    },
+    "alredySecureOtherSeedError": {
```
Comment (thesan) [inline apps/ledger-live-desktop/static/i18n/en/app.json:6458]:
```suggestion
    "alreadySecureOtherSeedError": {
```

**Review by thesan — COMMENTED** [review]
Just a few typos:

**Review by cgrellard-ledger — APPROVED** [review]

**Review by cgrellard-ledger — APPROVED** [review]

**Pull request merged**